content/hi/docs/home/_index.md

```diff
@@ -43,22 +43,22 @@ cards:
   title: "प्रशिक्षण"
   description: "कुबेरनेट्स में प्रमाणित हो कर अपने क्लाउड नेटिव प्रोजेक्ट सफल बनाएं!"
   button: "प्रशिक्षण देखें"
-  button_path: "/training"
+  button_path:/docs/reference
 - name: reference
   title: संदर्भ जानकारी देखें 
   description: शब्दावली, कमांड लाइन सिंटैक्स, API संसाधनो के प्रकार और टूल सेटअप करने के प्रलेखन। 
   button: संदर्भ देखें 
-  button_path: /docs/reference
+  button_path: /docs/contribute
```

Comment:
Here The button and description at line 49 and 50 indicate that it should redirect to the refernece page in hindi So here I think there is no need of changing button_path from /docs/refernece to /docs/contribute. 
The given path is correct.